windows/src/desktop/kmshell/main/KeyboardTIPCheck.pas

```diff
@@ -82,6 +82,7 @@ class function TKeyboardTIPCheck.CheckKeyboardTIPInstallStatus: Boolean;
       begin
         if kmcom.SystemInfo.IsAdministrator then
         begin
+          //TTIPMaintenance.DoRegister(k.ID, k.Languages[j].BCP47Code);
```

Comment:
Why the added line?  I'm guessing it's a development artifact?

Reply:
It's a hint to me that this is closer to the right direction for the fix (but not far enough).